src/Orleans/Streams/Internal/GenericAsyncObserver.cs

```diff
@@ -28,34 +28,34 @@ namespace Orleans.Streams
     /// <typeparam name="T">The type of object produced by the observable.</typeparam>
     internal class GenericAsyncObserver<T> : IAsyncObserver<T>
     {
-        private Func<T, StreamSequenceToken, Task> _NextAsync;
-        private Func<Exception, Task> _ErrorAsync;
-        private Func<Task> _CompletedAsync;
+        private Func<T, StreamSequenceToken, Task> onNextAsync;
+        private Func<Exception, Task> onErrorAsync;
+        private Func<Task> onCompletedAsync;
 
-        public GenericAsyncObserver( Func<T, StreamSequenceToken, Task> onNextAsync, Func<Exception, Task> onErrorAsync, Func<Task> onCompletedAsync )
+        public GenericAsyncObserver(Func<T, StreamSequenceToken, Task> onNext, Func<Exception, Task> onError, Func<Task> onCompleted)
```

Comment:
nit: argument names don't match names in extension functions, so argument names in exceptions won't match those in api.